AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreEndToEndTests.swift

```diff
@@ -180,31 +180,33 @@ class DataStoreEndToEndTests: SyncEngineIntegrationTestBase {
         let conditionalReceived = expectation(description: "Conditional save failed received")
 
         let hubListener = Amplify.Hub.listen(to: .dataStore) { payload in
-            guard let mutationEvent = payload.data as? MutationEvent
-                else {
-                    XCTFail("Can't cast payload as mutation event")
-                    return
-            }
-
-            // This check is to protect against stray events being processed after the test has completed,
-            // and it shouldn't be construed as a pattern necessary for production applications.
-            guard let post = try? mutationEvent.decodeModel() as? Post, post.id == newPost.id else {
-                return
-            }
+            if payload.eventName == HubPayload.EventName.DataStore.syncReceived || payload.eventName == HubPayload.EventName.DataStore.conditionalSaveFailed {
```

Comment:
I put a `if` statement here as a filter filtering `syncReceived` and `conditionalSaveFailed` events since we have added 9 more events.

Reply:
can you add new line to reduce line length?

Reply:
I believe `filter` can only filter on one event instead of multiple events.
Let me first use `guard` and add a new line

Reply:
looks like there's an `any` filter that you can pass multiple https://github.com/aws-amplify/amplify-ios/blob/main/Amplify/Categories/Hub/HubFilter.swift#L25

Reply:
Thanks. I didn't know this